tests/driver.c

```diff
@@ -197,13 +197,16 @@ int main(void) {
   ta_config_set(&ta_core.cache, &ta_core.service);
 
   printf("Total samples for each API test: %d\n", TEST_COUNT);
+  /*
```

Comment:
This doesn't help at all. You shall fix this. Otherwise, open a bug issue and resolve it in another PR.